### DISCUSSION TITLE
2 Nspanel - Blueprints

### DISCUSSION THREAD

**Discussion by ESANTOS74** [Blackymas/NSPanel_HA_Blueprint#2178] [Q&A]
Hi friends, as I liked the result of changing the flash from NSPANEL to ESPHOME I decided to buy another panel, it turns out that within HA Add the panel appears online, but it doesn't appear when I go to create a new bueprints automation.

**Comment by monzsca** [comment]
It may need to be added to home assistant, go to settings then devices and see if it's listed under discovered, if so, select configure and add it. I had the same problem and forgot to add it there.

**Reply by ESANTOS74** [reply]
Thanks friend, that's right, you can solve it this afternoon, in my case the discovery was automatic, but for some reason it didn't integrate directly, so I removed it and added it again, then it worked.
